harvard-university-of-worcester.csl

```diff
@@ -261,14 +261,14 @@
       </else>
     </choose>
   </macro>
-  <citation collapse="year" disambiguate-add-year-suffix="true" et-al-use-first="3" et-al-min="4">
+  <citation et-al-min="3" et-al-use-first="1" sort-separator="," disambiguate-add-year-suffix="true" collapse="year" cite-group-delimiter=" ">
```

Comment:
are you sure about the cite-group-delimiter? (Smith 1771 1776) is what that would give you. Looks odd, no? You probably want a comma here.